### PR TITLE
VACMS-000: Remove .git sub-directories after composer install & update

### PR DESCRIPTION
**Pull request by ElijahLynn**
## Description

This fixes that really annoying issue where `composer install` prompts you if you want to overwrite modified changes to certain dependencies/modules. We don't need the .git directories, unless we are working on an upstream module, and in that case we will likely clone it outside of the repo and symlink as needed.

## Testing done
Ran this locally with `composer install`. 

## Screenshots
image

## QA steps
1. `cd docroot/modules/cer`
2. Verify a `git status` returns git info about the `cer` module, not the CMS repo
3. Run `composer install`
4. Verify it finishes with a step that says "Removing .git sub-directories...`
5. Run `git status` again in the `cer` directory, you should see info about this branch instead

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Platform CMS Team`
- [ ] `Sitewide program`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
- [ ] Yes, and it's written in issue ____ and queued for publication.
  - [ ] Merge and ping the UX writer so they are ready to publish after deployment
- [ ] Yes, but it hasn't yet been written
  - [ ] Don't merge yet -- ping the UX writer to write and queue content
- [ ] No announcement is needed for this code change.
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

**Comment by ndouglas** [comment]
didn't nuke the content-build .git directory, didja?

**Review by swirtSJW — DISMISSED** [review]
This tests out.  I ran it locally and it removed the cer git instance


image

File: composer.json [hunk 1]
```diff
@@ -536,23 +537,24 @@
             "# This will allow you to run any command in the ./bin directory without a path.",
             "echo \"export PATH=${PATH}\""
         ],
+        "remove-git-dirs": [
+            "echo \"Removing .git sub-directories...\"",
+            "find . -mindepth 2 -type d -name .git | xargs rm -rf"
```
Comment (ndouglas) [inline composer.json:542]:
```suggestion
            "find . -mindepth 2 -type d -name .git -not -path './docroot/vendor/va-gov/content-build/.git' | xargs rm -rf"
```

Comment (ElijahLynn) [reply composer.json]: Thanks Nate, committed!

**Review by swirtSJW — APPROVED** [review]
Good catch on the scope to not include content_build @ndouglas 

Nice work @ElijahLynn

**Comment by swirtSJW** [comment]
image

**Pull request merged**